src/static/css/module/loan-comparison.less

```diff
@@ -157,73 +173,89 @@
     outline: 0;
     display: block;
   }
+  
+  .input-headers {
+    .respond-to-min(@tablet-min, {
+      padding-top: unit(@grid_gutter-width / @base-font-size-px, em);
+    });
+  }
+  
+  .interest-rate-update,
+  .interest-rate-loading {
+    display: none;
+  }
+  .update .interest-rate-update {
+    width: 100%;
+    display: block;
+    animation: fadein .5s;
+  }
+  
+  .updating .interest-rate-loading {
+    display: block;
+    width: 100%;
+    margin-left: 0;
+    height: 2em;
+    background-image: url(http://www.consumerfinance.gov/hmda/static/img/icon_spinner.gif);
+    background-attachment: scroll;
+    background-repeat: no-repeat;
+    background-position: center center;
+  }
+  .update,
+  .updating {
+    .interest-rate-content {
+      display: none;
+    }
+  }
+  
 }
 
 .input-headers {
-  .wrapper;
-
-  li {
-    .wrapper;
-    .grid_column(3);
-    text-align: center;
-    padding: 1em 0;
-  }
-
+  text-align: center;
+  
   h3 {
     .reset;
   }
 }
 
 /* labels and inputs are full width on small screens */
```

Comment:
Can these be full width on less than 400px width screens instead of 600? They look weird between 400-600:

![screen shot 2015-03-12 at 11 43 40 am](https://cloud.githubusercontent.com/assets/702526/6621472/388102f6-c8ad-11e4-8553-f815b41152c1.png)